pkg/core/state/nep5.go

```diff
@@ -45,6 +45,8 @@ type NEP5Transfer struct {
 // to the corresponding structures.
 type NEP5Balances struct {
 	Trackers map[util.Uint160]NEP5Tracker
+	// NextTransferBatch stores an index of the next transfer batch.
+	NextTransferBatch uint16
```

Comment:
`uint32`, please, we have some 28M transactions on mainnet, suppose that some 10% of them transfer something to/from one account, even with 128 elements batch it more than 20K batches, not that far away from 64K possible. Also note that it's just adds two bytes to a 22 byte key, no big deal, really, we have way longer keys already.